### PR TITLE
Update references to terraform templates

### PR DESCRIPTION
**Pull request by jfmyers9**
**NOTE**: This PR should not be merged until https://www.pivotaltracker.com/story/show/153669766 has been accepted and released in a final version.

[#153669766]

### CODE REVIEW

File: docs/installing/aws/deploying-bosh-aws.md [hunk 1]
```diff
@@ -31,23 +31,23 @@ $ export AWS_SECRET_ACCESS_KEY=dsfSDFKOSDFKOasdmasdKSADOK</p>
...
-	<p class="terminal">$ export private_subnet_ip_prefix="10.0.0.2"</p>
+1. Set the IP address prefix of the private subnet that will be used for CFCR VMs and the BOSH Director as an environment variable named `private_subnet_ip_prefix`. This prefix must be within the VPC CIDR range. For example:
+	<p class="terminal">$ export private_subnet_ip_prefix="10.0.1"</p>
 
 	!!! note
 		You create the public and private subnets with Terraform in a later step, so you do not need to create them in the AWS Console.
 
 1. Set the path of the Terraform state file as an environment variable named `kubo_terraform_state`. Enter the following command:
-	<p class="terminal">$ export kubo_terraform_state=~/terraform.tfstate</p>
+	<p class="terminal">$ export kubo_terraform_state="${HOME}/terraform.tfstate"</p>
```
Comment (jhvhs) [inline docs/installing/aws/deploying-bosh-aws.md:50]: I would suggest renaming the state file to `${HOME}/${prefix}.tfstate` to allow multiple bosh deployments from a single jumpbox.

Comment (jfmyers9) [reply docs/installing/aws/deploying-bosh-aws.md:50]: Yeah that makes sense.

**Comment by glestaris** [comment]
The tracker story has been accepted and the docs always use the latest deployment tar-ball so no need to wait for a new release to merge this. This PR can now be merged.

**Comment by mjgutermuth** [comment]
Thanks @jfmyers9 and @glestaris!

**Pull request merged**